### DISCUSSION TITLE
Related to customizing UI to handle signin

### DISCUSSION THREAD

**Discussion by MGW-SumathyV** [nocodb/nocodb#3055] [Q&A]
Hi,

I am calling nocodb signin API from my app server to sign in to nocodb .Once I receive the token in the response I am redirecting response in my server to load to projects list dashboard page.I am passing token as query param to this redirected URL.
I wanted to load projects dash board page directly without signin from UI. where should I modify the UI code to acheive this?

Thanks and Regards,
SumathyVenugopal

**Comment by MGW-SumathyV** [comment]
Hi ,
Could you please provide any update on this as its critical requirement for us.

**Reply by o1lab** [reply]
Sorry, if its a custom implementation - you will have to figure out this :)

**Reply by MGW-SumathyV** [reply]
ok fine
